samples/infrastructure.yaml

```diff
@@ -0,0 +1,55 @@
+## Kubernetes
+
+infrastructure:
+  name: prod-platformdemo-k8scluster
+  id: prodplatformdemok8scluster
+  description: ""
+  tags: {}
+  org: default
+  project: Platform_Demo
+  uses: kubernetes-gke| kubernetes-eks | kubernetes-aks | kubernetes-direct
```

Comment:
could these be simplified `gke`, `eks`, `aks` and `kubernetes`?  the thinking is that kubernetes is part of the acronym of these third party offerings so it is not needed as a prefix (e.g. `gke` is `google kubernetes engine`)

Reply:
yes we can do something like that.